src/main/scala/io/iohk/ethereum/consensus/ConsensusConfig.scala

```diff
@@ -72,14 +75,23 @@ object ConsensusConfig extends Logger {
     val blockCacheSize = config.getInt(Keys.BlockCacheSize)
     val miningEnabled = config.getBoolean(Keys.MiningEnabled)
     val optOut = config.getBoolean(Keys.TreasuryOptOut)
+    val checkpointPubKeys = readCheckpointPubKeys(config)
+    val minRequireSignatures = (Math.floor(checkpointPubKeys.length / 2) + 1).toInt
```

Comment:
As this is dependent on the checkpointPubKeys should we maybe define it as a function on the config object?